project/share/library_patches/arduino_mkrrgb/src/MKRRGBMatrix.h

```diff
@@ -0,0 +1,41 @@
+//
+// Created by danie on 2021-11-02.
+//
+
+#ifndef MKRRGBMATRIX_H
+#define MKRRGBMATRIX_H
+
+#include <cstddef>
+#include <cstdint>
+#include <ArduinoGraphics.h>
+#include <FramebufferAccess.h>
+#include "SMCE_dll.hpp"
+
+#define RGB_MATRIX_WIDTH 640
+#define RGB_MATRIX_HEIGHT 480
+#define PIXEL_FORMAT SMCE_Pixel_Format::RGB888
+#define BITS_PER_PIXEL 24
+#define FPS 60
```

Comment:
I doubt all of these are part of the MKRRGB API. If not, they should either be deleted, or prefixed with `SMCE__`